lib/minitest/reporters.rb

```diff
@@ -30,6 +30,11 @@ def self.use!(console_reporters = ProgressReporter.new, env = ENV, backtrace_fil
       Minitest.backtrace_filter = backtrace_filter unless backtrace_filter.nil?
 
       unless defined?(@@loaded)
+        # Ensure that minitest-reporters is initialized last, so that our patches to Minitest.reporters are not
+        # trampled by other Minitest plugins, like the core Rails 5 Minitest plugin.
+        Minitest.load_plugins
```

Comment:
what if plugins were already loaded?

Reply:
Minitest don't care: https://github.com/seattlerb/minitest/blob/master/lib/minitest.rb#L86

Reply:
Ok, let's try this.